comp110-worksheet-7-tests/UnitTest1.cs

```diff
@@ -8,70 +8,70 @@
 
 namespace comp110_worksheet_7_tests
 {
-	[TestFixture]
-	public class UnitTest1
-	{
-		string c_baseDirectory
-		{
-			get { return Path.Combine(TestContext.CurrentContext.TestDirectory, "..", "..", "pixelcarpack_kenney", "PNG"); }
-		}
+    [TestFixture]
```

Comment:
Try to avoid committing purely formatting changes (e.g. whitespace) to shared repos -- it can make the diff more difficult to read